k2/python/k2/__init__.py

```diff
@@ -66,6 +66,7 @@
 from .fsa_algo import linear_fsa
 from .fsa_algo import linear_fsa_with_self_loops
 from .fsa_algo import linear_fst
+from .fsa_algo import linear_fst_with_self_loops
```

Comment:
Please also make a change to icefall after merging this PR.

https://github.com/k2-fsa/icefall/blob/master/icefall/decode.py#L338
```python3
word_fsa_with_epsilon_loops = k2.linear_fsa_with_self_loops(word_fsa)
```